src/Cache/CacheOperationTypes/CacheOperationTypes.php

```diff
@@ -1020,6 +1024,11 @@ public function valuesArray(): array
         return $ret;
     }
 
+    public function fieldValueDictionary(): array
```

Comment:
I think by our naming conventions this would start with `value`.  I think maybe it would be `valueDictionary` but let's get @pgautier404 to weigh in.

Reply:
Yes, everything should begin with `value`. I thought we were going to go with something like `valueArray()` because dictionaries are arrays in PHP, but I could go with either name as long as it's consistent.

Reply:
See https://github.com/momentohq/client-sdk-php/pull/65#issuecomment-1315780503

Reply:
an https://github.com/momentohq/client-sdk-php/pull/65#issuecomment-1315789863